src/compile/legend/parse.ts

```diff
@@ -126,7 +126,10 @@ export function parseLegendForChannel(model: UnitModel, channel: NonPositionScal
 
   for (const property of LEGEND_COMPONENT_PROPERTIES) {
     if (
+      // Continuous and discrete gradients ignore symbol properties
       (legendType === 'gradient' && property.startsWith('symbol')) ||
+      (legendType === 'discrete' && property.startsWith('symbol')) ||
```

Comment:
Before I add any tests or open PRs in `vega`, I want to check if this is the behavior (a third legendType called discrete) we want as a public API, or if this was supposed to be a private/internal API.

The discrete legend type was added by @jheer in https://github.com/vega/vega/commit/ddd2bbff9897415be968fb60e2afa6c695c9eec4 , so I'll bring it up as a new issue in the vega repo.